var/spack/repos/builtin/packages/wrf/package.py

```diff
@@ -15,132 +60,309 @@ class Wrf(Package):
+            "em_squall2d_y",
+            "em_grav2d_x",
+            "em_seabreeze2d_x",
+            "em_scm_xy",
+        ),
+    )
 
-    variant('pnetcdf', default=True,
-            description='Parallel IO support through Pnetcdf libray')
+    variant(
+        "pnetcdf",
+        default=True,
+        description="Parallel IO support through Pnetcdf libray",
+    )
+
+    patch("patches/3.9/netcdf_backport.patch", when="@3.9.1.1")
+    patch("patches/3.9/tirpc_detect.patch", when="@3.9.1.1")
+    patch("patches/3.9/add_aarch64.patch", when="@3.9.1.1")
 
     # These patches deal with netcdf & netcdf-fortran being two diff things
     # Patches are based on:
     # https://github.com/easybuilders/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/w/WRF/WRF-3.5_netCDF-Fortran_separate_path.patch
-    patch('patches/4.0/arch.Config.pl.patch', when='@4.0')
-    patch('patches/4.0/arch.configure.defaults.patch', when='@4.0')
-    patch('patches/4.0/arch.conf_tokens.patch', when='@4.0')
-    patch('patches/4.0/arch.postamble.patch', when='@4.0')
-    patch('patches/4.0/configure.patch', when='@4.0')
-    patch('patches/4.0/external.io_netcdf.makefile.patch', when='@4.0')
-    patch('patches/4.0/Makefile.patch', when='@4.0')
-
-    patch('patches/4.2/arch.Config.pl.patch', when='@4.2')
-    patch('patches/4.2/arch.configure.defaults.patch', when='@4.2')
-    patch('patches/4.2/arch.conf_tokens.patch', when='@4.2')
-    patch('patches/4.2/arch.postamble.patch', when='@4.2')
-    patch('patches/4.2/configure.patch', when='@4.2')
-    patch('patches/4.2/external.io_netcdf.makefile.patch', when='@4.2')
-    patch('patches/4.2/var.gen_be.Makefile.patch', when='@4.2')
-    patch('patches/4.2/Makefile.patch', when='@4.2')
-
-    depends_on('mpi')
+    patch("patches/4.0/arch.Config.pl.patch", when="@4.0")
+    patch("patches/4.0/arch.configure.defaults.patch", when="@4.0")
+    patch("patches/4.0/arch.conf_tokens.patch", when="@4.0")
+    patch("patches/4.0/arch.postamble.patch", when="@4.0")
+    patch("patches/4.0/configure.patch", when="@4.0")
+    patch("patches/4.0/external.io_netcdf.makefile.patch", when="@4.0")
+    patch("patches/4.0/Makefile.patch", when="@4.0")
+    patch("patches/4.0/tirpc_detect.patch", when="@4.0")
+    patch("patches/4.0/add_aarch64.patch", when="@4.0")
+
+    patch("patches/4.2/arch.Config.pl.patch", when="@4.2")
+    patch("patches/4.2/arch.configure.defaults.patch", when="@4.2")
+    patch("patches/4.2/arch.conf_tokens.patch", when="@4.2")
+    patch("patches/4.2/arch.postamble.patch", when="@4.2")
+    patch("patches/4.2/configure.patch", when="@4.2")
+    patch("patches/4.2/external.io_netcdf.makefile.patch", when="@4.2")
+    patch("patches/4.2/var.gen_be.Makefile.patch", when="@4.2")
+    patch("patches/4.2/Makefile.patch", when="@4.2")
+    patch("patches/4.2/tirpc_detect.patch", when="@4.2")
+    patch("patches/4.2/add_aarch64.patch", when="@4.2")
+
+    depends_on("pkg-config", type=("build"))
```

Comment:
```suggestion
    depends_on("pkgconfig", type=("build"))
```
Spack has a virtual package that lets you use `pkg-config` or `pkgconf`